packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-line-settings.tsx

```diff
@@ -69,6 +71,29 @@ const LockedLineSettings = (props: Props) => {
     // Check if the line has length 0.
     const isInvalid = kvector.equal(point1.coord, point2.coord);
 
+    function getPrepopulatedAriaLabel() {
+        let str = `${capitalizeKind} from (${point1.coord[0]}, ${point1.coord[1]}) to (${point2.coord[0]}, ${point2.coord[1]})`;
+
+        if (labels && labels.length > 0) {
+            str += " with label";
+            // Make it "with labels" instead of "with label" if there are
+            // multiple labels.
+            if (labels.length > 1) {
+                str += "s";
+            }
+
+            for (let i = 0; i < labels.length; i++) {
+                // Separate additional labels with commas.
+                if (i > 0) {
+                    str += ",";
+                }
+                str += ` ${labels[i].text}`;
+            }
```

Comment:
Could this be simplified to

```suggestion
            str += ` ${labels.map((l) => l.text).join(", ")}`
```

or am I missing something?

Reply:
That's much cleaner, thank you!